pimd/pim_iface.c

```diff
@@ -1418,6 +1423,18 @@ int pim_if_gm_join_del(struct interface *ifp, pim_addr group_addr,
 			  " group %pPAs source %pPAs on interface %s",
 			  __func__, &group_addr, &source_addr, ifp->name);
 		return -3;
+	} else if (ij->join_type != join_type) {
```

Comment:
Since we have `if (!ij)…` that always returns.  Why do we need an elseif ?  There is no need for it.

Reply:
fixed